roofit/roofitcore/src/RooPlot.cxx

```diff
@@ -266,7 +266,45 @@ RooPlot::RooPlot(const RooAbsRealLValue &var, Double_t xmin, Double_t xmax, Int_
   _normBinWidth = (xmax-xmin)/nbins ;
 }
 
+////////////////////////////////////////////////////////////////////////////////
+//
+
+RooPlot* RooPlot::frame(const RooAbsRealLValue &var, Double_t xmin, Double_t xmax, Int_t nBins){
```

Comment:
Why this static function? Wouldn't it be better to call the constructor directly?
If the function is necessary, it should at least be documented for doxygen.

Reply:
This was mostly added to provide a consistent interface withe the other function defined in the next line.